src/poprox_recommender/components/joiners/concat.py

```diff
@@ -0,0 +1,9 @@
+from poprox_concepts import ArticleSet
+
+
+class Concatenate:
+    def __call__(self, candidates1: ArticleSet, candidates2: ArticleSet) -> ArticleSet:
+        reverse_articles = (candidates1.articles + candidates2.articles)[::-1]
+        articles = {article.article_id: article for article in reverse_articles}
+
+        return ArticleSet(articles=list(articles.values())[::-1])
```

Comment:
I think I see how this code works, but I don't think it's very obvious to someone who isn't familiar with ins and outs of dictionary ordering ordering and reversing. Maybe some comments describing why we build the dictionary in reverse order, and the reverse it again, would help?

Reply:
for consistency with the above change, this could also be `list(reversed(articles.values()))`.